src/main/java/com/odysseusinc/athena/service/mail/EmailSenderService.java

```diff
@@ -53,53 +51,34 @@
     private String from;
 
     @Autowired
-    public MailSender(JavaMailSender mailSender, MailContentBuilder contentBuilder) {
+    public EmailSenderService(MailContentBuilder contentBuilder, JavaMailSender mailSender) {
 
-        this.mailSender = mailSender;
         this.contentBuilder = contentBuilder;
+        this.mailSender = mailSender;
     }
 
-    public void send(AthenaUser user, Map<String, Object> parameters) throws MessagingException,
-            UnsupportedEncodingException {
-
-        send(user.getEmail(), parameters);
-    }
-
-    public void send(String email, Map<String, Object> parameters) throws MessagingException, MailException,
-            UnsupportedEncodingException {
+    public void send(EmailType messageType, Map<String, Object> parameters, String... emails) {
```

Comment:
Why not to move this to the EmailService? What benefits of separation?

Reply:
In next step of refactoring I plan to make sending email asynchronous, so this separation allows us to encapsulate rendering and sending in a separate thread with no direct dependency to any hibernate entity.

Reply:
Well, let's make an issue to the backlog with proposal of refactoring.

Reply:
@wivern - I have created  issue with the refactoring offer: https://github.com/OHDSI/Athena/issues/76